src/nsidc/metgen/readers/registry.py

```diff
@@ -17,8 +17,8 @@ def lookup(auth_id: str, extension: str) -> Callable[[str, Config], dict]:
             return function
 
     readers = {
-        ".nc": netcdf_reader.extract_metadata,
-        ".csv": csv.extract_metadata,
+        constants.NETCDF_SUFFIX: netcdf_reader.extract_metadata,
+        constants.CSV_SUFFIX: csv.extract_metadata,
```

Comment:
Thanks for doing this. I realized I left a lot of string literals laying around when my DUCk timer expired at the end of March. Sorry.